src/_guides/bump-sh-tutorials/huma.md

```diff
@@ -23,24 +23,182 @@ The following assumes your local machine is configured with Golang and you have
   ```
   This will run not only the application, but it will make API documentation available on <http://127.0.0.1:8000/>.
 
-4. Deploy your doc to Bump.sh with
+4. Deploy your doc to Bump.sh.
   ```bash
-  bump deploy http://127.0.0.1:8888/openapi.yaml \
+  bump deploy http://localhost:8888/openapi.yaml \
     --doc my-documentation-name \
     --token my-documentation-token
   ```
 
-If for any reason you are unable to download that `openapi.yaml` from a URL despite being able to access it, it's likely some odd network proxy settings on your machine or wifi. Grab it directly with wget or similar.
+## Export OpenAPI for Continuous Integration
 
+Running a web server and deploying that to Bump.sh might be ok locally, but if you'd like to automate everything properly it might become complicated to run an entire server instance on something like GitHub Actions. Let's skip that, and configure Huma to output OpenAPI via a CLI Command.
 
-```bash
-$ wget http://127.0.0.1:8888/openapi.yaml
+It's actually designed to do this, and whilst it's not documented currently there is [an example](https://github.com/danielgtaylor/huma/blob/main/examples/spec-cmd/main.go) in the codebase:
```

Comment:
Would you mind pointing to an immutable URL?

```suggestion
It's actually designed to do this, and whilst it's not documented currently there is [an example](https://github.com/danielgtaylor/huma/blob/3a7da129f4dccc8191b614206f63ebca1fcced1e/examples/spec-cmd/main.go) in the codebase:
```

(I usually always share “static” links of files on github instead of a file pointing to a moving reference - a branch, it's easy to do so when your on a file of a branch on github, just hit the 'y' key from the keyboard :slightly_smiling_face:)